tests/WE2E/machine_suites/coverage.hera.gnu.com

```diff
@@ -3,6 +3,7 @@ get_from_NOMADS_ics_FV3GFS_lbcs_FV3GFS
 grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR
 grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp_regional_plot
 grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta
+grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_RAP_suite_RAP_quilt_off
```

Comment:
Test should also be added to the "comprehensive" list

Reply:
@mkavulich I renamed it here (to `quilting_false`) and added it to `comprehensive`.